File: gestion-tickets/src/app/_models/TicketFilterRequest.ts
export interface TicketFilterRequest {
  userId?: number;
  clientId?: number;
  personnelId?: number;
  start?: string;       // ISO (ex. "2025-04-29")
  end?: string;
  granularity: 'daily' | 'weekly' | 'monthly' | 'yearly' | 'none';
  
}

File: GestionTicketsAPI/Helpers/TicketFilterRequest.cs
public class TicketFilterRequest
{
    public int? UserId      { get; set; }
    public int? ClientId { get; set; }
    public int? PersonnelId { get; set; }
    public DateTime? Start  { get; set; }
    public DateTime? End    { get; set; }
    public string  Granularity { get; set; } = "monthly";
}
